lib/index.js

```diff
@@ -16,96 +18,79 @@ exports.listTokens = listTokens
 exports.removeToken = removeToken
 exports.createToken = createToken
 
+const ProfileConfig = pudding({
+  creds: {},
+  hostname: {}
+})
+
 // try loginWeb, catch the "not supported" message and fall back to couch
-function login (opener, prompter, conf) {
+function login (opener, prompter, opts) {
```

Comment:
The conf/opts rename makes this diff a lot noiser and harder to read than it would otherwise be. It's the sort of thing that for that reason I think should be a separate commit.

Reply:
I've moved it into a separate commit